quadhoral-openmeteo/api_error.py

```diff
@@ -0,0 +1,39 @@
+'''This module contains the API error class'''
+
+class APIError(Exception):
```

Comment:
Similar to before, good use of error classes to categorise errors in an OOP fashion.

Reply:
I concur.